src/AstGen.zig

```diff
@@ -5370,20 +5370,20 @@ fn containerDecl(
     switch (token_tags[container_decl.ast.main_token]) {
         .keyword_struct => {
             const layout = if (container_decl.layout_token) |t| switch (token_tags[t]) {
-                .keyword_packed => std.builtin.Type.ContainerLayout.Packed,
-                .keyword_extern => std.builtin.Type.ContainerLayout.Extern,
+                .keyword_packed => std.builtin.Type.ContainerLayout.@"packed",
+                .keyword_extern => std.builtin.Type.ContainerLayout.@"extern",
                 else => unreachable,
-            } else std.builtin.Type.ContainerLayout.Auto;
+            } else std.builtin.Type.ContainerLayout.auto;
 
```

Comment:
Might be worth changing this to use inference while you're at it (`const layout: std.builtin.Type.ContainerLayout = ...`, `=> .@"packed"`, `=> .@"extern"`, `else .Auto`).

Reply:
Yeah 👍. This PR is still very much WIP at the moment as you can see by the breaking CI and draft status. Hope to finish this tonight though.

Reply:
Done